pkgs/applications/science/math/sage/python-modules/sage-setup.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage
+, sage-src
+, pkgconfig # the python module, not the pkg-config alias
```

Comment:
This is technically true, but the similar name is confusing to most people; it was added to `sagelib.nix` post-#116594, which landed after pkgconfig was already an alias. I understand the concern, but I personally don't think this pollutes the file and it may prevent future breakage. I'll make sure to remove the comment if/when this file gets moved outside the sage directory.